### DISCUSSION TITLE
Security inspected and took items from a character after giving items to another player

### DISCUSSION THREAD

**Discussion by Star-Tube** [FakeFishGames/Barotrauma#14177] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Handed non-stolen switches to another player in multiplayer. That player got searched immediately and got his items taken even though he had no stolen items on him. The base also continues to act strange after that point.

(I can provide a modlist if desired, ping me on discord at @star_tube)

### Reproduction steps

1. Start multiplayer campaign
2. Have two players logged in
3. Stand on a station
4. Hand an item from one player to the other using the new pass function
5. Observe angry npcs

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I gave this a quick test, but unfortunately couldn't find a way to reproduce the issue. The security doesn't seem to react in any way to giving items to other characters, so I suspect this requires some more specific circumstances to happen.

https://github.com/FakeFishGames/Barotrauma/assets/14351070/e8e0aa13-8dc8-4779-9b99-1056f1cdf16b
